dev-python/ipywidgets/ipywidgets-9999.ebuild

```diff
@@ -25,11 +25,9 @@ IUSE="test"
 RDEPEND="
 	dev-python/traitlets[${PYTHON_USEDEP}]
 	dev-python/ipykernel[${PYTHON_USEDEP}]
-	$(python_gen_cond_dep 'dev-python/wxpython:*[${PYTHON_USEDEP}]' python2_7)
 	"
 DEPEND="${RDEPEND}
 	test? (
-		$(python_gen_cond_dep 'dev-python/mock[${PYTHON_USEDEP}]' python2_7)
```

Comment:
Why is this line dropped? @flacjacket Since you added this dependency, maybe you could clarify?